gym/lib/gym/generators/package_command_generator_xcode7.rb

```diff
@@ -168,11 +168,15 @@ def config_content
         if $verbose
           UI.message("This results in the following plist file:")
           UI.command_output("-----------------------------------------")
-          UI.command_output(hash.to_plist)
+          UI.command_output(to_plist(hash))
           UI.command_output("-----------------------------------------")
         end
 
-        hash.to_plist
+        to_plist(hash)
+      end
+
+      def to_plist(hash)
```

Comment:
This looks great. Could you add a comment here on why we have this as a separate method and using the more complex way? This way we'll keep using this like it in the future 👍